app/controllers/idv/phone_question_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class PhoneQuestionController < ApplicationController
+    include ActionView::Helpers::DateHelper
+    include IdvStepConcern
+    include StepIndicatorConcern
+
+    before_action :confirm_verify_info_step_needed
+    before_action :confirm_agreement_step_complete
+    before_action :confirm_hybrid_handoff_needed, only: :show
+
+    def show
+      @title = t('doc_auth.headings.phone_question')
+    end
+
+    def confirm_agreement_step_complete
+      return if idv_session.idv_consent_given
+
+      redirect_to idv_agreement_url
+    end
+
+    def confirm_hybrid_handoff_needed
```

Comment:
Would it work to move `HybridHandoff#confirm_hybrid_handoff_needed` to IdvStepConcern and share it between the two controllers? I'm uneasy about the pattern of making the same checks, and then redirecting to HybridHandoff to let it redirect again. It's confusing code, which is okay if very temporary, but it also makes it more likely to run up against the limit of 10 redirects before the user gets a "too many redirects" failure.